### DISCUSSION TITLE
LineHinge():  'assigned to'  doesn't work

### DISCUSSION THREAD

**Discussion by SebastiaanVanHassel** [Dlubal-Software/RFEM_Python_Client#219] [Q&A] [answered]
Is there already a solution for the LineHinge() assigned_to problem
It does not work, unfortunately...

I try, with line 2 and surface 3 to create line hinge 1
LineHinge(1, '2/3')

**Comment by matchristmann** [comment, marked as answer]
Hello @SebastiaanVanHassel,

I've created an issue for this, you can find it under Issue Tab #224 .

Best Regards,
Matti
